requirements.txt

```diff
@@ -1,10 +1,10 @@
 # requirements
-Jinja2<=2.10.1
+Jinja2
 PyYAML<=5.2
 netaddr<=0.7.19
 prometheus_client
-git+https://github.com/juju/charm-helpers.git@87fc7ee5#egg=charmhelpers
+git+https://github.com/juju/charm-helpers.git@59f134b#egg=charmhelpers
 git+https://github.com/canonical/operator.git@0.8.0#egg=ops
-git+https://opendev.org/openstack/charm-ops-interface-ceph-client@cc10f29d4#egg=interface_ceph_client
+git+https://opendev.org/openstack/charm-ops-interface-ceph-client#egg=interface_ceph_client
 git+https://opendev.org/openstack/charm-ops-openstack#egg=ops_openstack
-git+https://opendev.org/openstack/charm-ops-interface-tls-certificates@2ec41b60#egg=ca_client
+git+https://opendev.org/openstack/charm-ops-interface-tls-certificates@f720af03a1
```

Comment:
As this is the master branch is there a need to pin to shas?  (it may be a workflow that I'm unfamiliar with in this charm!)

Reply:
Yes, we don't need to pin at all here. The interface_tls_certificates spec is wrong, though. We can use the following (I've tried it):
```
# requirements
markupsafe<=2.0.1
Jinja2
netaddr<=0.7.19
prometheus_client
git+https://github.com/juju/charm-helpers.git#egg=charmhelpers
git+https://github.com/canonical/operator.git#egg=ops
git+https://opendev.org/openstack/charm-ops-interface-ceph-client#egg=interface_ceph_client
git+https://opendev.org/openstack/charm-ops-openstack#egg=ops_openstack
git+https://opendev.org/openstack/charm-ops-interface-tls-certificates#egg=interface_tls_certificates
```

Reply:
No, this doesn't work as is. It breaks the capability to run FIO tests, like in the pull request in #6 , which required to integrate a fix in that very same pull request.
This cannot be seen easily because it breaks only when running the action and here is no CI, nor unit tests, for this charm to check truly if everything is fine. Moreover testing everything manually takes time.

The goal of this pull request is to make it work on Jammy based on the code, and before this it is also using SHAs.
And it was tested specifically on these commits, since these were the latest commit by the time of this pull request (nearly 6 months ago ...) and also to make sure nothing would break at least on both Focal and Jammy since we need to test manually for now that all features are working.

Considering that there are already multiple other pull requests on top waiting and modifying as well this file, I would prefer for this to be handled in a different pull request and tested when everything is merged.